### PR TITLE
feat: added image component

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

File: pages/components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+.img-xl{
+    width: 20rem;
+    height: 20rem;
+}
+
+.img-lg{
+    width: 15rem;
+    height: 15rem;
+}
```
Comment (shangkaul) [inline pages/components/Image/image.css:34]: you can add more image sizes + a class for square images

**Review by shangkaul — APPROVED** [review]
Apart from the suggestions, looks good to merge.

**Review by omkarpatke — APPROVED** [review]
I didn't see any suggestions so looks good to merge.

**Review by sri-rishi — APPROVED** [review]
Code and UI looks good to merge me

**Review by shangkaul — APPROVED** [review]
looks good to merge

**Pull request merged**